main/uart_nic.c

```diff
@@ -156,6 +179,7 @@ static int wifi_receive_cb(void *buffer, uint16_t len, void *eb) {
         }
     }
 
+    last_inbound_seen = now_seconds();
```

Comment:
Please move this up before the MAC filter. It will reduce the number of false-positive link down events.

Reply:
ACK